lsp-methods.el

```diff
@@ -1222,17 +1222,16 @@ interface TextDocumentEdit {
 
       (> start1 start2))))
 
-(define-inline lsp--apply-text-edits (edits)
+(defun lsp--apply-text-edits (edits)
   "Apply the edits described in the TextEdit[] object in EDITS."
-  (inline-quote
-    ;; We sort text edits so as to apply edits that modify earlier parts of the
-    ;; document first. Furthermore, because the LSP spec dictates that:
-    ;; "If multiple inserts have the same position, the order in the array
-    ;; defines which edit to apply first."
-    ;; We reverse the initial list to make sure that the order among edits with
-    ;; the same position is preserved.
-
-    (seq-do #'lsp--apply-text-edit (sort (nreverse ,edits) #'lsp--text-edit-sort-predicate))))
+  ;; We sort text edits so as to apply edits that modify latter parts of the
+  ;; document first. Furthermore, because the LSP spec dictates that:
+  ;; "If multiple inserts have the same position, the order in the array
+  ;; defines which edit to apply first."
+  ;; We reverse the initial list and sort stably to make sure the order among
+  ;; edits with the same position is preserved.
+  (mapc #'lsp--apply-text-edit
+        (sort (nreverse (seq-into edits 'list)) #'lsp--text-edit-sort-predicate)))
```

Comment:
Do you need to call `seq-into` here? IMO the functions (`mapc`, `sort`, `nreverse`) support sequences.

Reply:
Thanks.

Changed to `  (mapc #'lsp--apply-text-edit
        (sort (nreverse edits) #'lsp--text-edit-sort-predicate))
`